### ISSUE TITLE
Job bump-lockfile shouldn't push changes that are just metadata date updates

### ISSUE THREAD

**Issue by jlebon** [coreos/fedora-coreos-pipeline#232]
See e.g. https://github.com/coreos/fedora-coreos-config/commit/e34014c4812acb854a0acbd5e337096eddf51b6d:

```diff
diff --git a/manifest-lock.x86_64.json b/manifest-lock.x86_64.json                                                                                                                                                                           
index e1d65d0..91e58c5 100644                                                                                                                                                                                                               
--- a/manifest-lock.x86_64.json                                                                                                                                                                                                             
+++ b/manifest-lock.x86_64.json                                                                                                                                                                                                             
@@ -1088,7 +1088,7 @@                                                                                                                                                                                                                       
     }
   },
   "metadata": {
-    "generated": "2020-05-05T19:53:47Z",                                                                                                                                                                                                   
+    "generated": "2020-05-05T21:07:22Z",                                                                                                                                                                                                   
     "rpmmd_repos": {
       "fedora": {
         "generated": "2019-10-23T22:52:47Z"
```

We should just `cosa buildprep` from the latest build and use change detection the way the main pipeline does it.

**Comment by dustymabe** [comment]
@sohankunkerkar - would you want to work on this when you get free?

**Comment by dustymabe** [comment]
Is this fixed by #322. This "just metadata timestamp bump" commit just landed in `next-devel`: https://github.com/coreos/fedora-coreos-config/commit/c374504ad3fdc3d868dbeb273e05f43cb5c2afb0

**Comment by jlebon** [comment]
Yup, you're right. There's another subtle issue here. Will fix.
